.travis.yml

```diff
@@ -27,21 +25,20 @@ before_cache:
 # Exclude some combinations from build matrix. See:
 # https://docs.travis-ci.com/user/customizing-the-build/#Build-Matrix
 matrix:
-  exclude:
-    - scala: 2.11.12
-      jdk: oraclejdk9
-    - scala: 2.11.12
-      jdk: oraclejdk10
-    - scala: 2.11.12
-      jdk: oraclejdk11
+  include:
```

Comment:
We should later [use jabba](http://eed3si9n.com/all-your-jdks-on-travis-ci-using-jabba) to install AdoptOpenJDK instead.